### ISSUE TITLE
pre-train accuracy for fbwq_half

### ISSUE THREAD

**Issue by lihuiliullh** [malllabiisc/EmbedKGQA#116]
May I know how much the accuracy is when using kge trains fbwq_half ?
Also, when you use kgc to train fbwq_half, did you only use train.txt?

**Comment by lihuiliullh** [comment]
I used the pre-trained embedding on KGQA, the accuracy is very low. How did you learn the embedding use kge? @apoorvumang

**Comment by apoorvumang** [comment]
Can you please elaborate on what experiment you did, what commands you used?

**Comment by lihuiliullh** [comment]
I try to use kge to get the pre-train embedding of fbwq_half, and use the pre-trained embedding in multi-hop answering on fbwq_half. @apoorvumang

**Comment by apoorvumang** [comment]
Thanks for the details, let me try it with this config and I'll get back to you

**Comment by lihuiliullh** [comment]
Can you share your config file to me? so I can get the same performance in the paper @apoorvumang
